src/CMakeLists.txt

```diff
@@ -150,13 +152,15 @@ add_executable(phoebus
 
   tracers/tracers.hpp
   tracers/tracers.cpp
+
+  # Generated
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/generated>/compile_constants.hpp
 )
```

Comment:
Thanks for adding this! you might want to do the same thing for `test_utils.hpp` maybe cause tests are still failing.

Reply:
`test_utils` doesn't belong in `phoebuslib` but I re-added it to the relevant unit tests.